src/main/java/UtilsTest/PasswordsTest.java

```diff
@@ -0,0 +1,66 @@
+package UtilsTest;
+
+import Utils.Passwords;
+import Utils.Tuple;
+import jdk.nashorn.internal.ir.annotations.Ignore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PasswordsTest {
+
+    String password;
+
+    @BeforeEach
+    void setUp() {
+        password = "Password123";
+    }
+
+    @AfterEach
+    void tearDown() {
+    }
+
+    @Test
```

Comment:
Each @Test annotated function is a unit test. Personally I have been taught to name them, and organise them in a "Given, When, Then" layout. I think it's pretty readable.

**_Given_** a set of values
**_When_** something occurs
**_Then_** this should happen